docs/scripts/buildApi.ts

```diff
@@ -312,16 +312,39 @@ function findProperties(reflection: TypeDoc.DeclarationReflection) {
   return properties.sort((a, b) => a.name.localeCompare(b.name));
 }
 
-function extractEvents(eventsObject: TypeDoc.DeclarationReflection, apisToGenerate) {
-  const events: { name: string; description: string }[] = [];
+function extractEvents(
+  project: TypeDoc.ProjectReflection,
+  eventsObject: TypeDoc.DeclarationReflection,
+  apisToGenerate,
+) {
+  const events: { name: string; description: string; params?: string; event?: string }[] = [];
+  const eventLookupType = project.findReflectionByName(
+    'GridEventLookup',
+  )! as TypeDoc.DeclarationReflection;
   const allEvents = eventsObject.children!;
 
+  const eventLookup: { [eventName: string]: any } = {};
+  eventLookupType.children?.forEach((event) => {
+    const eventParams = {};
+    if (event.type && isReflectionType(event.type)) {
+      event.type.declaration.children?.forEach((eventProperty) => {
+        if (eventProperty.type) {
+          eventParams[eventProperty.name] = generateTypeStr(eventProperty.type);
+        }
+      });
+    }
+
+    eventLookup[event.name] = eventParams;
+  });
+
   allEvents.forEach((event) => {
     const description = linkify(event.comment?.shortText, apisToGenerate, 'html');
-
+    const eventProperties = eventLookup[event.escapedName!];
     events.push({
       name: event.escapedName!,
       description: renderMarkdownInline(description),
+      params: eventProperties.params,
+      event: eventProperties.event,
```

Comment:
You can use the `linkify` function and save the link to the interface instead of only the name.

Reply:
What do you want me to linkify here ?
The description is already linkified.